src/test/java/ca/bc/gov/backendstartapi/dto/ExampleDtoTest.java

```diff
@@ -5,57 +5,59 @@
 
 public class ExampleDtoTest {
```

Comment:
The style guide says that every public class must have its Javadoc (see https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required).

Reply:
Great point. Thanks.